addons/hr/views/hr_plan_views.xml

```diff
@@ -180,5 +180,25 @@
                 </form>
             </field>
         </record>
+
+        <record id="mail_activity_type_form_inherit" model="ir.ui.view">
+            <field name="name">mail.activity.type.form.inherit</field>
+            <field name="model">mail.activity.type</field>
+            <field name="inherit_id" ref="mail.mail_activity_type_view_form"/>
+            <field name="arch" type="xml">
+                <label for="delay_count" position="replace">
+                    <label for="delay_count" attrs="{'invisible': ['|', '&amp;', ('chaining_type', '=', 'suggest'), ('suggested_next_type_ids', '=', []), '&amp;', ('chaining_type', '=', 'trigger'), ('triggered_next_type_id', '=', False)]}"/>
+                </label>
+                <field name="delay_count" position="replace">
+                    <field class="oe_inline pe-1 o_input_3ch" name="delay_count" attrs="{'invisible': ['|', '&amp;', ('chaining_type', '=', 'suggest'), ('suggested_next_type_ids', '=', []), '&amp;', ('chaining_type', '=', 'trigger'), ('triggered_next_type_id', '=', False)]}"/>
+                </field>
+                <field name="delay_from" position="replace">
+                    <field class="oe_inline ps-1 pe-2" name="delay_unit" attrs="{'invisible': ['|', '&amp;', ('chaining_type', '=', 'suggest'), ('suggested_next_type_ids', '=', []), '&amp;', ('chaining_type', '=', 'trigger'), ('triggered_next_type_id', '=', False)]}"/>
+                </field>
+                <field name="delay_unit" position="replace">
+                    <field class="oe_inline" name="delay_from" attrs="{'invisible': ['|', '&amp;', ('chaining_type', '=', 'suggest'), ('suggested_next_type_ids', '=', []), '&amp;', ('chaining_type', '=', 'trigger'), ('triggered_next_type_id', '=', False)]}"/>
+                </field>
```

Comment:
it's better to use `position="attributes"` instead so that if other attributes are changed on the base view, it's not overriden here :)
```suggestion
                <label for="delay_count" position="attributes">
                    <attribute name="attrs">
                        {'invisible': ['|', '&amp;', ('chaining_type', '=', 'suggest'), ('suggested_next_type_ids', '=', []), '&amp;', ('chaining_type', '=', 'trigger'), ('triggered_next_type_id', '=', False)]}
                    </attribute>
                </label>
                <field for="delay_count" position="attributes">
                    <attribute name="attrs">
                        {'invisible': ['|', '&amp;', ('chaining_type', '=', 'suggest'), ('suggested_next_type_ids', '=', []), '&amp;', ('chaining_type', '=', 'trigger'), ('triggered_next_type_id', '=', False)]}
                    </attribute>
                </field>
                <field for="delay_from" position="attributes">
                    <attribute name="attrs">
                        {'invisible': ['|', '&amp;', ('chaining_type', '=', 'suggest'), ('suggested_next_type_ids', '=', []), '&amp;', ('chaining_type', '=', 'trigger'), ('triggered_next_type_id', '=', False)]}
                    </attribute>
                </field>
                <field for="delay_unit" position="attributes">
                    <attribute name="attrs">
                        {'invisible': ['|', '&amp;', ('chaining_type', '=', 'suggest'), ('suggested_next_type_ids', '=', []), '&amp;', ('chaining_type', '=', 'trigger'), ('triggered_next_type_id', '=', False)]}
                    </attribute>
                </field>
```